vsdkx/addon/zoning/settings.py

```diff
@@ -0,0 +1,6 @@
+DEFAULT = {
+    "filter_class_ids": [],
```

Comment:
I would rename this to `class_ids` and update it also on the `processor.py` & `README.md`:
```suggestion
    "class_ids": [],
```

Reply:
I agree with you on this, but this requires more changes than that. we'd also have to make changes in settings.yaml on zoning compatible processors and all their configs, in visionx & client configs. 

If we want to do that, I presume we'd have to open separate issue or maybe even technical story on our board to track all the required changes

Reply:
Let's just leave it as is for the time being and circle back about it if we see fit in the future.